### ISSUE TITLE
High cpu load due to large number of interrupts

### ISSUE THREAD

**Issue by yul** [tobiasBora/MT7630E_3.16#5]
I noticed that after enabling this driver my laptop's cooler is not stopping, top shows that ksoftirqd process takes 100% CPU. 

Numbers in cat /proc/interrupts for network card IRQ18 raise for 10+ each second, currently look like this:

```
           CPU0       CPU1       CPU2       CPU3       CPU4       CPU5       CPU6       CPU7       
 18:       3214      25405         19        239         36        237         23         14  IR-IO-APIC-fasteoi   0000:03:00.0, 0000:03:00.0
```

Kernel version is 3.16.0-46-generic #62~14.04.1-Ubuntu SMP Tue Aug 11 16:27:16 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

It happens even if I'm not connected to any network, only shutting down network interface stops it.
I'm not a linux pro, so not sure what to check next, any help appreciated.

**Comment by yul** [comment]
I've tried this version: https://github.com/lwfinger/mt7630 
It seems to be working ok for me.

**Comment by tobiasBora** [comment]
I tried this version, and indeed I do not have any 100% CPU load, but the Wifi doesn't detect any network around... Did do you do anything else that "make; sudo make install; sudo ./load.sh" to load the module ?

**Comment by tobiasBora** [comment]
I also try something like:

```
$ sudo ifconfig wlan0 up
SIOCSIFFLAGS: Erreur d'entrée/sortie
```

(I can translate it with Input/Output error)

**Comment by yul** [comment]
no, perhaps you have unsupported kernel version? That driver works almost ok for me, except sometimes it starts disconnecting from wi-fi, until poweroff/poweron (reboot not helping), but it happens pretty rare.

**Comment by tobiasBora** [comment]
And what did you de to enable it at startup ?

On 1 septembre 2015 07:44:05 HAEC, Yuri Lopukhov notifications@github.com wrote:

> no, perhaps you have unsupported kernel version? That driver works
> almost ok for me, except sometimes it starts disconnecting from wi-fi,
> until poweroff/poweron (reboot not helping), but it happens pretty
> rare.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/tobiasBora/MT7630E_3.16/issues/5#issuecomment-136592084

## 

Envoyé de mon téléphone Android avec K-9 Mail. Excusez la brièveté.

**Comment by yul** [comment]
I just installed it with sudo make install

**Comment by tobiasBora** [comment]
It's pretty strange, when I do that it doesn't load at startup, I need to manually run load.sh…
